src/foundry/client-esm/dice/_module.d.mts

```diff
@@ -8,6 +8,6 @@ export * as types from "./_types.mjs";
 export * as terms from "./terms/_module.mjs";
 
 export { default as Roll } from "./roll.mjs";
-export { default as RollGrammar } from "./grammar.pegjs";
+// export { default as RollGrammar } from "./grammar.pegjs"; // TODO: Figure out if there's a good way to statically type the PEGGY grammar
```

Comment:
As soon as https://github.com/peggyjs/peggy/pull/538 is in I'll type `grammar.pegjs`.